packages/uiweb/src/lib/components/space/SpacesUI.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import SpaceBanner, { SpaceBannerProps } from './SpaceBanner';
+import SpaceFeed, { SpaceFeedProps } from './SpaceFeed';
+import SpaceWidget, { SpaceWidgetProps } from './SpaceWidget';
+import SpaceTrendingList, { SpaceTrendingListProps } from './SpaceTrendingList';
+import { useSpaceData } from '../../context';
+import { SignerType } from '../../types';
+import { ENV } from '../../config';
+
+interface SpacesUIProps {
```

Comment:
```suggestion
interface ISpacesUIProps {
```

suggestion: interface names should be leading with an `I` so that it helps us identify it is a TS Interface. This helps in avoiding ambiguity between Interfaces and local component props.